src/AzureClient/Visualization/TargetStatusEncoders.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Azure.Quantum.Client.Models;
+using Microsoft.Jupyter.Core;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Quantum.IQSharp.AzureClient
+{
+    internal static class TargetStatusExtensions
+    {
+        internal static Dictionary<string, object> ToDictionary(this TargetStatus target) =>
+            new Dictionary<string, object>()
+            {
+                { "targetName", target.Id },
```

Comment:
I think this should be 'targetId'.
We are also considering adding a "Friendly name" to the target metadata that can be used as name later.

Reply:
Thanks, that's great to know. I'll make a pass through and consistently call this "id" everywhere.